lib/rspec/sidekiq/matchers/base.rb

```diff
@@ -196,12 +208,58 @@ def on(queue)
           self
         end
 
+        def once
+          set_expected_count :exactly, 1
+          self
+        end
+
+        def twice
+          set_expected_count :exactly, 2
+          self
+        end
+
+        def thrice
+          set_expected_count :exactly, 3
+          self
+        end
+
+        def exactly(n)
+          set_expected_count :exactly, n
+          self
+        end
+
+        def at_least(n)
+          set_expected_count :at_least, n
+          self
+        end
+
+        def at_most(n)
+          set_expected_count :at_most, n
+          self
+        end
+
+        def times
+          self
+        end
+        alias :time :times
+
+        def set_expected_count(relativity, n)
+          n =
+            case n
+            when Numeric then n
+            when :once   then 1
+            when :twice  then 2
+            when :thrice then 3
```

Comment:
⚠️  Add an `else` clause to be safe (and avoid unintentionally setting `n` to `nil`) Perhaps default should be coercing `n` to an Integer?